src/db/schema.ts

```diff
@@ -38,16 +54,12 @@ export const chatTable = sqliteTable("chat", {
   unread_count: int().notNull().default(0),
   is_group: int().notNull().default(0),
   ...timestamps
-});
+})
```

Comment:
these are unwanted conflicts that i dont want to resolve 
please mute your plugin